pkg/api/azenvtypes.go

```diff
@@ -93,9 +93,9 @@ var (
 		EtcdDownloadURLBase:              "mcr.microsoft.com/oss/etcd-io/",
 		KubeBinariesSASURLBase:           "https://acs-mirror.azureedge.net/kubernetes/",
 		WindowsTelemetryGUID:             "fb801154-36b9-41bc-89c2-f4d4f05472b0",
-		CNIPluginsDownloadURL:            "https://acs-mirror.azureedge.net/cni/cni-plugins-amd64-" + CNIPluginVer + ".tgz",
-		VnetCNILinuxPluginsDownloadURL:   "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-" + AzureCniPluginVerLinux + ".tgz",
-		VnetCNIWindowsPluginsDownloadURL: "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
+		CNIPluginsDownloadURL:            "https://acs-mirror.azureedge.net/cni-plugins/" + CNIPluginVer + "/binaries/cni-plugins-linux-amd64-" + CNIPluginVer + ".tgz",
```

Comment:
This url is now downloadable.  Since we're not chaing cni pluign, I suggest we don't touch this
 
wget https://acs-mirror.azureedge.net/cni-plugins/v0.7.6/binaries/cni-plugins-linux-amd64-v0.7.6.tgz
--2020-05-19 13:14:47--  https://acs-mirror.azureedge.net/cni-plugins/v0.7.6/binaries/cni-plugins-linux-amd64-v0.7.6.tgz
Resolving acs-mirror.azureedge.net (acs-mirror.azureedge.net)... 2600:1405:a000::172d:2e83, 2600:1405:a000::172d:2e9a, 104.86.182.43, ...
Connecting to acs-mirror.azureedge.net (acs-mirror.azureedge.net)|2600:1405:a000::172d:2e83|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-05-19 13:14:48 ERROR 404: Not Found.

Reply:
or you change CNIPluginVer = "v0.7.6" to 0.8.5

But then we will need to update VHD again with this 0.8.5 cni

Reply:
reverting back but asking @tamilmani1989

Reply:
Yeah those are standard plugins we can do those in a different change.